spoon-decompiler/pom.xml

```diff
@@ -0,0 +1,112 @@
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>fr.inria.gforge.spoon</groupId>
+        <artifactId>spoon-pom</artifactId>
+        <version>7.2.0-SNAPSHOT</version>
+        <relativePath>../spoon-pom</relativePath>
+    </parent>
+
+    <groupId>fr.inria.gforge.spoon</groupId>
+    <artifactId>spoon-decompiler</artifactId>
+    <packaging>jar</packaging>
+    <version>0.1.0-SNAPSHOT</version>
+    <name>Spoon Bytecode</name>
+    <description>Extension of spoon for bytecode through decompilation</description>
+    <url>http://spoon.gforge.inria.fr/</url>
+
+    <dependencies>
+        <dependency>
+            <groupId>fr.inria.gforge.spoon</groupId>
+            <artifactId>spoon-core</artifactId>
+            <version>${spoon.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/org.jboss.windup.decompiler.fernflower/fernflower -->
+        <dependency>
+            <groupId>org.jboss.windup.decompiler.fernflower</groupId>
+            <artifactId>fernflower</artifactId>
+            <version>2.5.0.Final</version>
+        </dependency>
+        <dependency>
+            <groupId>org.benf</groupId>
+            <artifactId>cfr</artifactId>
+            <version>0.135</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <version>2.4</version>
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <mainClass>spoon.Launcher</mainClass>
+                        </manifest>
+                    </archive>
+                    <descriptorRefs>
+                        <descriptorRef>jar-with-dependencies</descriptorRef>
+                        <!-- descriptorRef>project</descriptorRef -->
+                    </descriptorRefs>
+                    <!-- descriptor>src/main/assembly/project.xml</descriptor -->
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>make-assembly</id><!-- this is used for inheritance merges -->
+                        <phase>package</phase><!-- append to the packaging phase. -->
+                        <goals>
+                            <goal>attached</goal><!-- goals == mojos -->
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
```

Comment:
you removed checkstyle from spoon-core, so I assume you put it in spoon-pom, so it means you don't need it here.

Reply:
No. Both `spoon-core` and `spoon-decompiler` declare the checkstyle plugin but with differents option (exculsion of some sources for exemple).

Reply:
But the checkstyle profile is now in `spoon-pom`.